### ISSUE TITLE
App-Bedienungshilfen

### ISSUE THREAD

**Issue by JanneckLange** [digital-scouts/dpsg-nami-app#138]
https://docs.flutter.dev/ui/accessibility-and-internationalization/accessibility

- VoiceOver (iOS) / TalkBack (Android) 
> The screen reader should be able to describe all controls on the page when you tap on them, and the descriptions should be intelligible. 

- Sprachsteuerung (iOS)
- Texte
  - Fetter Text (iOS)
  - Größerer Text (iOS / Android)
 
> The UI should remain legible and usable at very large scale factors for text size and display scaling.

> All tappable targets should be at least 48x48 pixels.

- Ausreichender Kontrast (Umkehren Intelligent iOS)

> Controls should be usable and legible in colorblind and grayscale modes.

> We encourage you to have a contrast ratio of at least 4.5:1 between controls or text and the background, with the exception of disabled components. Images should also be vetted for sufficient contrast.
